core/statistics/machine/netStatistics_test.go

```diff
@@ -2,24 +2,26 @@ package machine
 
 import (
 	"fmt"
+	"sync/atomic"
 	"testing"
 
 	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/shirou/gopsutil/net"
 	"github.com/stretchr/testify/assert"
 )
 
 func TestNetStatistics(t *testing.T) {
 	t.Parallel()
 
-	net := &NetStatistics{}
+	ns := &NetStatistics{}
 
-	net.ComputeStatistics()
-	bpsRecv := net.BpsRecv()
-	bpsSent := net.BpsSent()
-	bpsRecvPeak := net.BpsRecvPeak()
-	bpsSentPeak := net.BpsSentPeak()
-	bpsRecvPercent := net.PercentSent()
-	bpsSentPercent := net.PercentRecv()
+	ns.ComputeStatistics()
+	bpsRecv := ns.BpsRecv()
+	bpsSent := ns.BpsSent()
+	bpsRecvPeak := ns.BpsRecvPeak()
+	bpsSentPeak := ns.BpsSentPeak()
+	bpsRecvPercent := ns.PercentSent()
+	bpsSentPercent := ns.PercentRecv()
 
 	fmt.Printf("Recv: %s/s, sent: %s/s\n", core.ConvertBytes(bpsRecv), core.ConvertBytes(bpsSent))
```

Comment:
can you please refactor this test because this test has no `asserts` just print some information

Reply:
right

Reply:
removed the unnecessary test and moved the prints in the next one

Reply:
refactored, added more unit tests